kythe/go/serving/tools/write_tables/write_tables.go

```diff
@@ -64,17 +67,32 @@ var (
 
 	experimentalBeamPipeline = flag.Bool("experimental_beam_pipeline", false, "Whether to use the Beam experimental pipeline implementation")
 	beamShards               = flag.Int("beam_shards", 0, "Number of shards for beam processing. If non-positive, a reasonable default will be chosen.")
+	beamK                    = flag.Int("beam_k", 0, "Amount of memory to use when creating level DB shards.")
+	beamInternalSharding     intListFlag
 	experimentalColumnarData = flag.Bool("experimental_beam_columnar_data", false, "Whether to emit columnar data from the Beam pipeline implementation")
 	compactTable             = flag.Bool("compact_table", false, "Whether to compact the output LevelDB after its creation")
 )
 
 func init() {
+	flag.Var(&beamInternalSharding, "beam_internal_sharding", "Internal sharding for tuning performance")
```

Comment:
Be more specific in the flag description

Reply:
Updated the description. Hopefully this makes sense.